JSONAPI.TodoMVC.API/Startup.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Data.Entity;
+using System.Reflection;
+using System.Web;
+using System.Web.Http;
+using Autofac;
+using Autofac.Integration.WebApi;
+using JSONAPI.ActionFilters;
+using JSONAPI.Core;
+using JSONAPI.EntityFramework.ActionFilters;
+using JSONAPI.Json;
+using JSONAPI.TodoMVC.API.Controllers;
+using JSONAPI.TodoMVC.API.Models;
+using Microsoft.Owin;
+using Owin;
+
+namespace JSONAPI.TodoMVC.API
+{
+    public class Startup
+    {
+        private const string DbContextKey = "TodoMVC.DbContext";
+
+        private readonly Func<IOwinContext, TodoMvcContext> _dbContextFactory;
+
+        public Startup()
+            : this(context => new TodoMvcContext())
+        {
+            
+        }
+
+        public Startup(Func<IOwinContext, TodoMvcContext> dbContextFactory)
+        {
+            _dbContextFactory = dbContextFactory;
+        }
+
+        public void Configuration(IAppBuilder app)
+        {
+            // Setup db context for use in DI
+            app.Use(async (context, next) =>
```

Comment:
Would it be terribly non-standard to rename `context` to `owinContext` or `appContext` or something in this scope? When I first started reviewing this I came here and was completely turned around (not knowing OWIN) because I thought `context` *was* a `DbContext` (this was common naming practice in EF4 days...which was probably also bad).